docs/galapagos_user_guide.md

```diff
@@ -0,0 +1,460 @@
+# Galapagos User Guide
+
+## Log in / First access
+
+When first opening Galapagos, you need to authenticate yourself. You will see 
+a small window asking you for your keycloak credentials. Here you can enter 
+a valid combination of username and password. 
+
+![](galapagos_user_guide_images/login.png)
+
+After successfully entering your username and password, you will be redirected to
+Galapagos and land on the start screen.
+
+![](galapagos_user_guide_images/start-screen.png)
+
+You can already find several elements here:
+
+  * The menu on the left. Here you can jump to the main functions of Galapagos.
+  * The info bar above. Here you can change the current Kafka environment and also change the language of the user interface. 
+  * The individual pages, which are discussed in more detail in the following sections.
+
+## Get basic and connection information
+
+On the dashboard (see screenshot above) you will find the following elements:
+
+  * Information about the current Kafka cluster.You can use the "Bootstrap Servers" e.g. in your
+    properties file for the Kafka connection in your application (Kafka property `bootstrap.servers`).
+  * Useful start links. 
+  * The "Latest Changes" on the selected Kafka cluster.
+  * The Galapagos version.
+  
+  The first thing you will find under the start links (as shown in the image below based on `spring` framework) is the so-called "Kafka Truststore". This is a 
+  Java keystore (JKS format) that contains the (self-signed) root certificates of the individual Kafka environments 
+  as "Trusted Certificates". If you integrate this truststore via your Kafka properties (standard Kafka properties 
+  `ssl.truststore.location` and `ssl.truststore.password`, password = changeit), your Kafka client trusts the Kafka 
+  clusters when establishing the connection.
+  
+  ![](galapagos_user_guide_images/config-template.png)
+  
+  Please note that you also need a client certificate to establish a successful connection! Information on this can
+  be found below in the section "Receive a client certificate".
+
+## Search Topics
+
+  To search for existing "API-Topics" (Topics universally visible and readable on-subscribe, see section "Subscribe to topic") on the current Kafka environment go to the menu "Browse Topics".
+  The following screen appears:
+  
+  ![](galapagos_user_guide_images/browse-topics.png) 
```

Comment:
Images are too big. Better use plain HTML (img tag) with width specification - main display medium will be GitHub.